### ISSUE TITLE
option to only sort defined keys

### ISSUE THREAD

**Issue by baskaal** [hugoattal/eslint-plugin-sort-keys-custom-order#5]
This plugin is awesome, thanks for making it. There is one little thing that prevents me from using it. Wanted to share because I don't believe im the only one with this usecase.

Besides the keys defined in the `orderedKeys` option, all other keys get sorted alphabetically by default. I wanted to use this to only sort CSS in JS objects by a specific order. So maybe just one extra option where you can toggle this off would be helpful imo.

**Comment by hugoattal** [comment]
Done and available in 2.1.0 !

```json
{
    "sort-keys-custom-order/object-keys": [
        "error", 
        { 
            "orderedKeys": ["id","name","title"],
            "sorting": "none"
        }
    ]
}
```

**Comment by baskaal** [comment]
damn thats quick, thanks so much! 👌🏼
